### ISSUE TITLE
Some hotkeys are intercepted by OS when setting up

### ISSUE THREAD

**Issue by phillijw** [deskflow/deskflow#7856]
### Project

Deskflow

### Deskflow version number

1.17.0+r1

### Build type

Deskflow package (downloaded from Deskflow)

### Operating systems (OS)

- [X] Windows
- [ ] macOS
- [ ] Linux (X11)
- [ ] Linux (Wayland)
- [ ] BSD-derived
- [ ] Other (please specify)

### Wayland on Linux

- [ ] I have reviewed the Wayland [known issues](https://github.com/deskflow/deskflow/discussions/7499) and my issue is new
- [ ] I am not using Wayland on Linux

### Signing on macOS

- [ ] I have authorized the app to run on my Mac
- [ ] I am not using macOS

### Continuous build

- [ ] I have tried the latest continuous build and the issue persists
- [ ] I am unable to try the latest continuous build

### OS versions/distros

- Windows 11

### Deskflow configuration

- Irrelevant in this case

### What steps will reproduce the problem?

1. Configure
2. Hotkeys
3. New
4. Click the "keystroke" button
5. Hit the "Print Screen" key

Instead of it identifying this key, it just takes a screenshot on my machine. It would be nice if I could have the option to enter the key name manually instead of relying on me pressing it. Doing it through the configuration file is a pain in the booty

**Comment by sithlord48** [comment]
I don't think we can do anything about that one. Some key(s) and combos are grabbed by the OS before we can even get a chance to use them print screen is one of them

**Comment by phillijw** [comment]
> I don't think we can do anything about that one. Some key(s) and combos are grabbed by the OS before we can even get a chance to use them print screen is one of them

Does that mean if I added it to my configuration file directly it wouldn't send the key to the other machine?

**Comment by sithlord48** [comment]
not sure i have not tried to do at but if you do try please report back on if it worked or not
